internal/runners/languages/languages.go

```diff
@@ -78,15 +78,12 @@ func (l *Languages) Run() error {
 	}
 
 	// Fetch resolved artifacts list for showing full version numbers.
```

Comment:
I don't think this comment applies anymore. What do you think about updating it?